app/views/admin/users/new.html.slim

```diff
@@ -5,23 +5,23 @@ div
     .form-inputs
       = f.label :first_name, "First Name"
       = f.text_field :first_name
-      
+
       = f.label :last_name, "Last Name"
       = f.text_field :last_name
-      
+
       = f.label :email, "E-Mail Address"
       = f.text_field :email
-      
+
       = f.label :roles, "Roles"
       - User::ROLES.each do |role|
         .checkboxes
-          = check_box_tag 'user[roles][#{role}]', role, @user.roles.include?(role), {:name => "user[roles][]"}
+          = check_box_tag "user[roles][#{role}]", role, @user.roles.include?(role), {name: 'user[roles][]'}
```

Comment:
The string interpolation wasn't taking effect here because of the single quotes. It didn't actually break the back-end role assignment functionality, but it made it so clicking the label didn't check the box. Relatively minor.